src/include/daos/drpc_modules.h

```diff
@@ -55,6 +55,7 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_BIO_HEALTH_QUERY	= 210,
 	DRPC_METHOD_MGMT_SMD_LIST_DEVS		= 211,
 	DRPC_METHOD_MGMT_SMD_LIST_POOLS		= 212,
+	DRPC_METHOD_MGMT_POOL_GET_ACL		= 213,
```

Comment:
Since we don't have container ACLs at the moment this is not for right now but we might want to think about what this interface would look like if we want to allow the admins to not just set pool ACLs but also container ACLs from the admin client as well.

Reply:
Good thought - I was thinking about that when designing the Protobuf messages. IMO it should be a completely separate dRPC command. On the handler side the behavior will be very different to look up the container. We can reuse the PB messages, though, since input will be a UUID and output will be an ACL (hence the generic GetACLReq/GetACLResp naming).